ext/web/02_event.js

```diff
@@ -917,13 +917,16 @@
   class EventTarget {
     constructor() {
       this[eventTargetData] = getDefaultTargetData();
+      this[webidl.brand] = webidl.brand;
     }
 
     addEventListener(
       type,
       callback,
       options,
     ) {
+      const self = this ?? globalThis;
```

Comment:
I have a feeling like the global `addEventListener` methods are not bound correctly if this is necessary.